drivers/clock_control/Kconfig.stm32wb

```diff
@@ -0,0 +1,65 @@
+# STM32WB PLL configuration options
+
+# Copyright (c) 2019 Linaro
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_STM32WBX
+
+config CLOCK_STM32_PLL_M_DIVISOR
+	int "PLL divisor"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
+	default 1
+	range 1 8
+	help
+	  PLL divisor, allowed values: 1-8.
+	  With this, ensure that the PLL VCO input ranges from 2.66 to 16MHz.
+
+config CLOCK_STM32_PLL_N_MULTIPLIER
+	int "PLL multiplier"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
+	default 20
+	range 6 127
+	help
```

Comment:
can you check if PLLN with 8 ≤ PLLN ≤ **86**

Reply:
As discussed, this correspond to latest Reference Manual, and it matches with CubeMX implementation.
Nevertheless I updated L5 PLLN which was a bit ambigus in Reference Manual.